src/sorting/python/heap_sort/README.md

```diff
@@ -1,4 +1,4 @@
-Merge sort implementation
+Heap sort implementation
 Author: Vlad421
 
 Implementation of heap sort algorithm
```

Comment:
Well this is definately should not be here!!
You made corrections to heap_sort staying in the linked_list branch!!

Reply:
Are it didn't push? Or I fixed it on another brunch....